README.md

```diff
@@ -0,0 +1,16 @@
+# TrackIt
+TrackIt is a inventory tracking management system.
+
+## Usage
+### Running the server on a local machine
+- this example assumes all package dependencies have already been installed - see below for details about install package dependencies.
+```shell
```

Comment:
have the npm install before npm run dev. that way you don't need the assumes all packages statement. also this is the way to run the project in development mode.